### PR TITLE
(#83) Moving Get to main package

### PR DESCRIPTION
**Pull request by scristalli**
This is for #83.

As requested in the issue, I moved the `Get` class from the `test` package to the `main` package.

### CODE REVIEW

**Comment by 0crat** [comment]
@scristalli[/z](https://www.0crat.com/u/scristalli) this pull request is too small, just 0 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @llorllale[/z](https://www.0crat.com/u/llorllale) please review this and merge or reject

**Comment by llorllale** [comment]
@scristalli missing tests for this class

**Comment by scristalli** [comment]
@llorllale I added a test case (only one since there is only the constructor to test)

File: src/test/java/org/cactoos/http/GetTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        new Assertion<>(
+            "Must build GET request correctly from URI",
+            new TextOf(
+                new Get(
+                    new URI(
+                        "http://host/path/to/resource"
+                    )
+                )
+            ),
+            new TextHasString("GET /path/to/resource HTTP/1.1\r\nHost:host")
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/GetTest.java]: @scristalli I believe `Host:` should be followed by a single space, and then the value

Comment (scristalli) [reply src/test/java/org/cactoos/http/GetTest.java]: @llorllale I agree. I edited the `Get` class and the test accordingly.

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos-http#91` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Pull request merged**